### PR TITLE
Add first atoum tests

### PR DESCRIPTION
**Pull request by vonglasow**
Adding tests

### CODE REVIEW

File: Bin/Assert.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
  *
  * New BSD License
  *
- * Copyright © 2007-2014, Ivan Enderlin. All rights reserved.
+ * Copyright © 2007-2015, Ivan Enderlin. All rights reserved.
```
Comment (jubianchi) [inline Bin/Assert.php]: `s/2007-2015/2007-2016/` everywhere ;)

Comment (jubianchi) [inline Bin/Assert.php]: The attributions should also be fixed: https://github.com/hoaproject/Consistency/blob/master/Prelude.php#L11

File: Context.php [hunk 2]
```diff
@@ -113,8 +113,14 @@ public function offsetGet ( $id ) {
         if($value instanceof DynamicCallable)
             return $value($this);
 
-        if(true === is_callable($value))
+        if(true === is_callable($value)) {
```
Comment (jubianchi) [inline Context.php]: this `if` and the next one could be grouped in a single one `if`

File: Ruler.php [hunk 3]
```diff
@@ -120,6 +120,20 @@ public static function interprete ( $rule ) {
...
+     *
+     * @access      public
+     * @param       string  $rule    Rule.
+     * @return      \Hoa\Ruler\Model
+     * @throw       \Hoa\Ruler\Exception
+     * @deprecated  Will be removed in 2.x.
+     */
+    public static function interprete ( $rule ) {
+
+        return static::interpret($rule);
```
Comment (jubianchi) [inline Ruler.php]: do we trigger an `E_DEPRECATED` ?

File: Test/Unit/Documentation.php [hunk 4]
```diff
@@ -165,11 +165,42 @@ public function case_compile ( ) {
 RESULT;
 
         $this
-            ->when($result = LUT::interprete(
```
Comment (jubianchi) [inline Test/Unit/Documentation.php]: As the old `interprete` ethod still exists, I think we should test both `interprete` and `interpret` and check that their results are stricly equal.

**Comment by jubianchi** [comment]
todo: 
* minor things to fix/check 
* a rebase

:heart:

**Comment by Hywan** [comment]
ping

**Comment by Hywan** [comment]
I guess we can close this PR now 😄.

**Pull request closed without merging**